hydroflow_datalog_core/src/lib.rs

```diff
@@ -708,16 +695,22 @@ fn apply_aggregations(
                 }
                 TargetExpr::Aggregation(Aggregation::CountUnique(..)) => {
                     let idx = syn::Index::from(agg_idx);
-                    after_group_lookups
-                        .push(parse_quote_spanned!(get_span(field.span)=> a.#idx.unwrap().1));
+                    after_group_lookups.push(parse_quote_spanned!(get_span(field.span)=> a.#idx.1));
```

Comment:
Seems like `after_group_lookups` would be a lot better as a iterator than a for loop